index.html

```diff
@@ -42,79 +44,231 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(1).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Multi-Post Stories</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(2).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Tonic</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio(3).png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Multi-Post Stories</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
+        </ul>
+        <button type="button" class="see_button">See project</button>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="header-text1">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-media-icons1">
```

Comment:
- [ ] Kindly let us add a left margin to this container to match the design.

- Your design
<img width="113" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227908173-36c10061-9129-49f7-a2a5-2413f3c8e8e0.png">

- Figma design

<img width="119" alt="right" src="https://user-images.githubusercontent.com/41163571/227708184-9c2f18d8-f186-4724-9078-b26ed1990926.png">